pkg/config/remote/api/http.go

```diff
@@ -118,3 +120,44 @@ func (c *HTTPClient) Fetch(ctx context.Context, request *pbgo.LatestConfigsReque
 
 	return response, err
 }
+
+// FetchOrgData org data
+func (c *HTTPClient) FetchOrgData(ctx context.Context) (*pbgo.OrgDataResponse, error) {
+	url := c.baseURL + orgDataEndpoint
+	log.Debugf("Querying url %s", url)
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, url, &bytes.Buffer{})
+	if err != nil {
+		return nil, fmt.Errorf("failed to create request: %w", err)
+	}
+	req.Header = c.header
+
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return nil, fmt.Errorf("failed to issue request: %w", err)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
		return nil, fmt.Errorf("failed to issue org data request: %w", err)
```